src/Modules/Data/Commands.Data/NewPowerBITable.cs

```diff
@@ -0,0 +1,54 @@
+﻿/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Management.Automation;
+using Microsoft.PowerBI.Common.Api.Datasets;
+using Microsoft.PowerBI.Common.Client;
+
+namespace Microsoft.PowerBI.Commands.Data
+{
+    [Cmdlet(CmdletVerb, CmdletName)]
+    [OutputType(typeof(Table))]
+    public class NewPowerBITable : PowerBIClientCmdlet
```

Comment:
For cmdlets that don't need the client, you could extend PowerBICmdlet directly.

Reply:
Okay, done.